PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCacheRenderers/PBMMediaViewDelegate.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PBMMediaViewDelegate.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+@objc public protocol PBMMediaViewDelegate {
+    func onMediaPlaybackStarted(_ mediaView: MediaView)
```

Comment:
Need to add `View` for each method name -> `onMediaView***`.  Because it is a delegate for MediaView not for Media.